src/engine/enginemaster.cpp

```diff
@@ -513,22 +519,22 @@ void EngineMaster::process(const int iBufferSize) {
 
         // Submit master samples to the side chain to do shoutcasting, recording,
```

Comment:
ok maybe something from this comment can go where that pointer is declared.  "m_pPostProcessChain" maybe is a better name?  Still not sure why it needs to be a double pointer rather than just pointing to one or the other buffer